05/index.html

```diff
@@ -0,0 +1,132 @@
+    />
+    <title>Document</title>
+  </head>
+  <body>
+    <section class="section section__navbar">
+      <div class="container container__navbar">
+        <nav class="navbar">
+          <img
+            class="navbar__logo"
+            src="assets/img/menu-logo.PNG"
+            alt="menu-logo"
+          />
+          <input type="checkbox" id="navbar__hamburger--toggle" />
+          <label class="navbar__hamburger" for="navbar__hamburger--toggle">
+            &#9776;
+          </label>
+          <ul class="navbar__list">
+            <li class="navbar__list-item">
+              <a class="navbar__list-link" href="#">Food</a>
+            </li>
+            <li class="navbar__list-item">
+              <a class="navbar__list-link" href="#">Pictures</a>
+            </li>
+            <li class="navbar__list-item">
+              <a class="navbar__list-link" href="#">History</a>
+            </li>
+            <li class="navbar__list-item">
+              <a class="navbar__list-link" href="#">Events</a>
+            </li>
+          </ul>
+        </nav>
+      </div>
+    </section>
+    <section class="section section__1">
+      <div class="container container__section-1">
+        <div class="section-1__container">
+          <img
+            class="section-1__img"
+            src="https://static01.nyt.com/images/2024/01/10/multimedia/AS-Burrito-vzhk/AS-Burrito-vzhk-master768.jpg?quality=75&auto=webp"
+            alt="burrito"
+          />
+        </div>
+        <div class="section-1__container">
+          <img
+            class="section-1__img"
+            src="https://ichef.bbci.co.uk/food/ic/food_16x9_832/recipes/smoky_tofu_tacos_35909_16x9.jpg"
+            alt="Taco"
+          />
+        </div>
+      </div>
+    </section>
+    <section class="section section__2">
+      <div class="container container__section-2">
+        <div class="section-2__container">
+          <h2 class="section-2__title">Mexican Food</h2>
+          <p class="section-2__text">
+            Mexican food is a vibrant cuisine known for its bold flavors and
+            rich cultural heritage. Key ingredients include corn, beans, chili
+            peppers, avocados, and various meats. Iconic dishes like tacos,
+            filled with diverse ingredients, and mole, a complex sauce often
+            served with meat, highlight this cuisine's diversity. Regional
+            specialties, such as Oaxacan mole and Yucatán achiote dishes,
+            showcase unique flavors. Street food, including elote (grilled corn)
+            and quesadillas, is integral to the culture. Overall, Mexican
+            cuisine combines indigenous traditions with Spanish and other
+            influences, making it beloved worldwide for its deliciousness and
+            communal dining experiences.
+          </p>
+        </div>
+        <div class="section-2__container">
+          <h2 class="section-2__title">Burrito</h2>

```

Comment:
Raczej staraj się bardziej określić co to za element zamiast pisać 1, 2 itp. - za miesiąc czy dwa te cyferki nic nie będą znaczyć. Co gorsza może się okazać że ich kolejność będzie do zmiany i wtedy będzie to wprowadzać niepotrzebny chaos.